starknet_py/contract_test.py

```diff
@@ -119,24 +118,6 @@ def test_compute_address_throws_on_no_source():
     )
 
 
-def test_transaction_hash():
```

Comment:
So there's no possibility to compute tx hash beforehand right now?

Reply:
There is an option. Invoke function has the "hash" property so it is possible